src/raven.js

```diff
@@ -413,6 +413,16 @@ Raven.prototype = {
             timestamp: now() / 1000
         }, obj);
 
+        if (isFunction(this._globalOptions.breadcrumbCallback)) {
+            var result = this._globalOptions.breadcrumbCallback(crumb);
+
+            if (result) {
+                crumb = result;
+            } else {
```

Comment:
What do we expect returning a falsey, non `false` value to do? Does it stick the falsey value into the breadcrumbs array? Would you expect a falsey value to use the original crumb that was passed into the callback? 

I could also imagine throwing if we don't get an object or `false` since it means the callback isn't built correctly. However, throwing in raven is obviously problematic since that is what tracks exceptions. Throwing in the breadcrumb handler though seems theoretically more okay than other places.